auth_app/templates/auth_app/home.html

```diff
@@ -0,0 +1,106 @@
+{% extends "base.html" %}
+{% load bootstrap4 %}
+{% bootstrap_css %}
+{% block page_content %}
+
+<div class="container-fluid">
+
+    <!-- Page Heading -->
+<h1 class="h3 mb-2 text-gray-800">Welcome to EDA Games!</h1>
+<div style="white-space: pre">
+    Hello again! we are very excited that you are here participating in our selection process. We want you to enjoy it as much as we do. The most important thing 
+    is that you be constant and always try to improve yourself.
+
+</div>
+<div align='center'>
+    <font color=031473>
+    <em>
+'If you can't see an example of what you could be,
+<br>
+you really aren't going to have that extra incentive to break through any types of barriers.'
+<br>
+- Julia Hartz. CEO of Evenbrite -
+    </em>
+    </font>
+</div>
+<div style="white-space: pre">
```

Comment:
nit: this style could be a class. That allows you to use the same style in a different component.

Reply:
Yes, when I was working on my branch I create a class on CSS file, and then I tried to use it here... but I don´t know why it didn't work... then I use another class only for test if it was an HTML problem or a problem with my class, and it was second. So, I decided to use it this way (knowing it could be a better way to be implemented).